app/actors/curation_concerns/actors/add_as_member_of_collections_actor.rb

```diff
@@ -0,0 +1,23 @@
+module CurationConcerns
+  module Actors
+    class AddAsMemberOfCollectionsActor < AbstractActor
+      def create(attributes)
+        collection_ids = attributes.delete(:member_of_collection_ids)
+        next_actor.create(attributes) && add_to_collections(collection_ids)
+      end
+
+      def update(attributes)
+        collection_ids = attributes.delete(:member_of_collection_ids)
+        add_to_collections(collection_ids) && next_actor.update(attributes)
```

Comment:
Is there a reason for `create` that `next_actor.create` happens before `add_to_collections` but for `update` the `next_actor.update` happens after the `add_to_collections`?